helpers/logging_config.py

```diff
@@ -1,19 +1,41 @@
 import json
 from copy import deepcopy
 
-from celery._state import get_current_task
 from pythonjsonlogger.jsonlogger import JsonFormatter
 
 from helpers.environment import Environment, get_current_env
 
+task_name = None
+task_id = None
+
+# this is a temporary hack to get the task information working in the logs again
```

Comment:
Unless there's a GitHub ticket to fix this for good there are good chances that this "temporary hack" will become permanent